src/comps/DevEnvSettings.tsx

```diff
@@ -0,0 +1,21 @@
+import {
+  MaxFeeAmountView,
+  SetBitcoinDUrl,
+  SetEmilyUrl,
+  SetSeedPhraseForDeposit,
+  SetSignerPubkey,
+} from "./Deposit";
+
+const DevEnvSettings = () => {
+  return (
+    <>
+      <SetSignerPubkey />
+      {/* <SetSeedPhraseForDeposit /> */}
```

Comment:
Should we keep, eventually feature-gated, a "dummy" wallet that we can set via seed phrase for testing without having to use external wallets?